### PR TITLE
Add lint-changelog-yaml command

### PR DESCRIPTION
**Pull request by felixfontein**
First step to deprecating antsibull-lint; see https://github.com/ansible-community/antsibull/issues/410.

### CODE REVIEW

File: src/antsibull_changelog/cli.py [hunk 1]
```diff
@@ -553,6 +568,23 @@ def lint_fragments(config: ChangelogConfig, fragments: List[ChangelogFragment],
...
+    """
+    errors = lint_changelog_yaml(
+        args.changelog_yaml_path, no_semantic_versioning=args.no_semantic_versioning)
+
+    messages = sorted(set(f'{error[0]}:{error[1]}:{error[2]}: {error[3]}' for error in errors))
+
+    for message in messages:
+        print(message)
+
+    return 3 if messages else 0
```
Comment (briantist) [inline src/antsibull_changelog/cli.py:585]: Where are the return codes defined? Might be nice to have constants for readability (not for this PR, just a general suggestion).

File: changelogs/fragments/76-lint-changelog-yaml.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "Add ``changelogs/changelog.yaml`` file format linting subcommand that was previously part of antsibull-lint (https://github.com/ansible-community/antsibull-changelog/pull/76)."
```
Comment (briantist) [inline changelogs/fragments/76-lint-changelog-yaml.yml:2]: Might be good to also link to the associated discussion you opened. unless you don't want that in the changelog.

**Review by briantist — APPROVED** [review]
Overall looks good to me, would like to see some tests that also check for the lint failing on files with problems, but that could be an additional PR.

**Comment by felixfontein** [comment]
There are already quite a few tests for the linting code (see https://github.com/ansible-community/antsibull-changelog/blob/main/tests/functional/test_changelog_yaml_linter.py and the subdirectories `bad` and `good`). There is no functional test for the new subcommand though, I'll add that later.

File: src/antsibull_changelog/cli.py (hunk 1 above)
Comment (felixfontein) [reply src/antsibull_changelog/cli.py:585]: There is no definition. The only rule right now is that 0 means success, and anything else error.

File: changelogs/fragments/76-lint-changelog-yaml.yml (hunk 2 above)
Comment (felixfontein) [reply changelogs/fragments/76-lint-changelog-yaml.yml]: Done.

File: src/antsibull_changelog/cli.py (hunk 1 above)
Comment (felixfontein) [reply src/antsibull_changelog/cli.py:585]: I've created #77 for this.

**Review by briantist — APPROVED** [review]

**Comment by felixfontein** [comment]
@briantist thanks for reviewing!

**Pull request merged**